fixtures/fun-generation/led-pot-12x1w-qcl-rgb-ww-40.json

```
@@ -0,0 +1,507 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Fun Generation LED Pot 12x1W QCL RGB WW 40°",
```

```suggestion
  "name": "LED Pot 12x1W QCL RGB WW 40°",
```

I think if it's in the "fun-generation" folder, we don't need "Fun Generation" in the title, too.